ocs_ci/utility/uninstall_openshift_logging.py

```diff
@@ -0,0 +1,75 @@
+"""
+Function to teardown the openshift-logging
+"""
+import logging
+
+from ocs_ci.ocs import constants, ocp
+
+logger = logging.getLogger(__name__)
+
+
+def uninstall_cluster_logging():
+    """
+    Function to uninstall cluster-logging from the cluster
+    1. Deletes the project "openshift-logging"
+        & "openshift-operators-redhat"
+    2. Deletes the CSC resource "cluster-logging-operator"
+        & "elasticsearch"
+    """
+
+    # Deleting the clusterlogging
+    clusterlogging_obj = ocp.OCP(
+        kind='ClusterLogging', namespace='openshift-logging'
+    )
+    assert clusterlogging_obj.delete(resource_name='instance')
+
+    # Validating all the pods are in Terminating state
+    pod_obj = ocp.OCP(
+        kind=constants.POD, namespace='openshift-logging'
+    )
+    ret = pod_obj.get()
+    assert ret, "Pods are in Running state."
```

Comment:
I don't think this is the right way to validate pods are in in terminating state because ret always gets the output like apiVersion, items. So the above statement never asserts